### PR TITLE
EC-313 Make views for routes 

### PR DESCRIPTION
**Pull request by jhnsmth**

### CODE REVIEW

File: app/controllers/TestController.scala [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+  def showSettingsForm(): Action[AnyContent] = Action { implicit request: Request[AnyContent] =>
+    Ok(views.html.settings(settingsForm.fill(TestController.SettingsData(wallet.flatMap(w => Try(lsmStorage.getWalletSecret(w)).toOption).map(_.privKeyBytes).map(Base58.encode).getOrElse("")))))
+  }
+
+  def showWalletInfo() = Action.async { implicit request =>
+    wallet match {
+      case Some(w) => {
+        val balance: Future[Option[Long]] = ws.loadWalletInfo(w).map(_.balance).map(Some(_)).recover { case NonFatal(_) => None }
+        val hash: String = Base16.encode(Blake2b256.hash(AccountLockedContract(w.account).bytes))
```
Comment (jhnsmth) [inline app/controllers/TestController.scala]: @oskin1 pls have a look

File: app/controllers/TestController.scala [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    wallet match {
+      case Some(w) => Ok(views.html.payment(Base16.encode(w.pubKey), paymentTransactionRequestForm))
+      case None => Redirect(routes.TestController.message("You should set up a wallet before making transaction"))
+    }
+  }
+
+  def sendPaymentTransactionFromForm(walletId: String): Action[AnyContent] = Action.async {
+    implicit request =>
+      paymentTransactionRequestForm.bindFromRequest.fold(
+        errors => Future.successful(Redirect(routes.TestController.message("Wrong transaction parameters\n" + errors.errors.mkString("\n") + errors))), //Future.successful(Ok(views.html.payment("", errors))),
```
Comment (GusevTimofey) [inline app/controllers/TestController.scala]: remove comment

File: app/controllers/TestController.scala [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        ptr => ts.sendPaymentTransaction(walletId, ptr).map {
+          _.status match {
+            case StatusCodes.OK => Redirect(routes.TestController.message("The transaction has been successfully sent to blockchain"))
+            case errorCode => Redirect(routes.TestController.message("Sending trasaction to the node has failed with code: " + errorCode)) //InternalServerError("Explorer node is down")
```
Comment (GusevTimofey) [inline app/controllers/TestController.scala]: remove comment

File: app/controllers/TestController.scala [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+          }
+        }.recover {
+          case e: IllegalArgumentException => Redirect(routes.TestController.message(e.getMessage))
+          case e: ExecutionException => Redirect(routes.TestController.message(e.getCause.getMessage + "\n" + e.getCause.getStackTrace.mkString("\n"))) // BadRequest(e.getMessage)
```
Comment (GusevTimofey) [inline app/controllers/TestController.scala]: remove comment

File: app/controllers/TestController.scala [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    wallet match {
+      case Some(w) => Ok(views.html.scripted(Base16.encode(w.pubKey), scriptedTransactionRequestForm))
+      case None => Redirect(routes.TestController.message("You should set up a wallet before making transaction"))
+    }
+  }
+
+  def sendScriptedTransactionFromForm(walletId: String): Action[AnyContent] = Action.async {
+    implicit request =>
+      scriptedTransactionRequestForm.bindFromRequest.fold(
+        errors => Future.successful(Redirect(routes.TestController.message("Wrong transaction parameters\n" + errors.errors.mkString("\n") + errors))), //Future.successful(Ok(views.html.payment("", errors))),
```
Comment (GusevTimofey) [inline app/controllers/TestController.scala]: remove comment

File: app/controllers/TestController.scala [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+        str => ts.sendScriptedTransaction(walletId, str).map {
+          _.status match {
+            case StatusCodes.OK => Redirect(routes.TestController.message("The transaction has been successfully sent to blockchain"))
+            case errorCode => Redirect(routes.TestController.message("Sending trasaction to the node has failed with code: " + errorCode)) //InternalServerError("Explorer node is down")
```
Comment (GusevTimofey) [inline app/controllers/TestController.scala]: remove comment

File: app/controllers/TestController.scala [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+          }
+        }.recover {
+          case e: IllegalArgumentException => Redirect(routes.TestController.message(e.getMessage))
+          case e: ExecutionException => Redirect(routes.TestController.message(e.getCause.getMessage + "\n" + e.getCause.getStackTrace.mkString("\n"))) // BadRequest(e.getMessage)
```
Comment (GusevTimofey) [inline app/controllers/TestController.scala]: remove comment

File: app/views/layout.scala.html [hunk 8]
```diff
@@ -0,0 +1,39 @@
+@*
```
Comment (GusevTimofey) [inline app/views/layout.scala.html]: comments

File: app/controllers/TestController.scala [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+import play.api.data.Forms._
+import play.api.i18n.I18nSupport
+import scorex.crypto.encode.{Base16, Base58}
+import models._
+import scorex.crypto.hash.Blake2b256
+import services.{TransactionService, WalletService}
+import storage.LSMStorage
+
+@Singleton
+class TestController @Inject()(implicit ec: ExecutionContext, ts: TransactionService, ws: WalletService, lsmStorage: LSMStorage, cc: ControllerComponents) extends AbstractController(cc) with I18nSupport {
```
Comment (GRYE) [inline app/controllers/TestController.scala]: Give a class more meaningful name

File: app/controllers/TestController.scala [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+  var wallet: Option[Wallet] = None
```
Comment (GRYE) [inline app/controllers/TestController.scala]: wallet should not be stored in controller, but rather be requested from corresponding service (and it should not be var)

**Pull request merged**